packages/suite/src/support/messages.ts

```diff
@@ -7816,6 +7816,18 @@ export default defineMessages({
     },
     TR_COINJOIN_STEP_3_DESCRIPTION: {
         id: 'TR_COINJOIN_STEP_3_DESCRIPTION',
-        defaultMessage: 'Your Bitcoin will be mixed with other peoples’ to make it untracable. ',
+        defaultMessage: 'Your Bitcoin will be mixed with other peoples’ to make it untracable.',
+    },
+    TR_COINJOIN_CEX_WARNING: {
+        id: 'TR_COINJOIN_CEX_WARNING',
+        defaultMessage: 'Exchanges may not serve you if you use CoinJoin',
+    },
+    TR_COINJOIN_CEX_EXPLANATION: {
+        id: 'TR_COINJOIN_CEX_EXPLANATION',
+        defaultMessage: 'They may reject your deposits or withdrawals and blacklist you',
+    },
+    TR_I_UNDERSTAND: {
+        id: 'TR_I_UNDERSTAND',
```

Comment:
Please do not use content of the message as the ID. Changing of the ID is hard and if the content change to something else like "Dismiss" it will be unnecessarily confusing.

Reply:
I just did it not to block the PR by so small change